cekit-helpers/cekit-cache-server/src/main/java/org/kie/cekit/cacher/utils/BuildUtils.java

```diff
@@ -0,0 +1,226 @@
+package org.kie.cekit.cacher.utils;
+
+import com.fasterxml.jackson.core.Version;
+import org.kie.cekit.cacher.builds.yaml.YamlFilesHelper;
+import org.kie.cekit.cacher.objects.PlainArtifact;
+import org.kie.cekit.cacher.properties.CacherProperties;
+import org.kie.cekit.image.descriptors.module.Module;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.lang.invoke.MethodHandles;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@ApplicationScoped
+public class BuildUtils {
+
+    private final Logger log = Logger.getLogger(MethodHandles.lookup().lookupClass().getName());
+
+    // artifacts and zip file names used on Nightly and CR builds
+    public String RHDM_ADD_ONS_DISTRIBUTION_ZIP = "rhdm_add_ons_distribution.zip";
+    public String RHDM_ADD_ONS_ZIP = "rhdm-%s-add-ons.zip";
+
+    public String RHDM_DECISION_CENTRAL_DISTRIBUTION_ZIP = "rhdm_decision_central_distribution.zip";
+    public String RHDM_DECISION_CENTRAL_EAP7_DEPLOYABLE_ZIP = "rhdm-%s-decision-central-eap7-deployable.zip";
+
+    public String RHDM_KIE_SERVER_DISTRIBUTION_ZIP = "rhdm_kie_server_distribution.zip";
+    public String RHDM_KIE_SERVER_EE8_ZIP = "rhdm-%s-kie-server-ee8.zip";
```

Comment:
For RHDM are missing nightly zip constants

Reply:
updated :)